web-server/src/components/AnimatedInputWrapper/slider.module.css

```diff
@@ -0,0 +1,29 @@
+.animationWrapper{
+    position: absolute;
```

Comment:
you are using `absolute` position, but the parent (element wrapping this component) isn't having `relative` position.
this can cause unexpected behaviour when you are trying to debug.

Reply:
Gave `position:relative` to `.placeholder` which was the parent element of `.animationWrapper`